lib/flood_risk_engine/engine.rb

```diff
@@ -12,12 +12,32 @@ class Engine < ::Rails::Engine
       app.config.i18n.load_path += Dir[config.root.join("config/locales/**/**/", "*.{rb,yml}").to_s]
     end
 
+    # Automatically add our migrations into the main apps migrations
+    initializer :append_migrations, before: :load_config_initializers do |app|
+      unless app.root.to_s.match root.to_s
+        config.paths["db/migrate"].expanded.each do |expanded_path|
+          app.config.paths["db/migrate"] << expanded_path
+        end
+      end
+    end
+
```

Comment:
If the migrations are pulled into the host app as described in [Getting Started With Engines](http://guides.rubyonrails.org/engines.html#engine-setup), we should not have to do anything else to get them in.

That is, the rake task `rake flood_risk_engine:install:migrations` will pull the migrations from the engine and install them into the host app.